clearwater-infrastructure/usr/share/clearwater/bin/reload_default_ifcs_config

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# @file reload_default_ifcs_config
+#
+# Project Clearwater - IMS in the Cloud
+# Copyright (C) 2017 Metaswitch Networks Ltd
```

Comment:
Need to switch this out for the new copyright text
(same for second file as well)